### PR TITLE
adding rest client tests for jbpm-quarkus-example

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

File: jbpm-quarkus-example/pom.xml [hunk 1]
```diff
@@ -63,6 +60,16 @@
       <artifactId>quarkus-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-rest-client</artifactId>
+      <scope>test</scope>
```
Comment (mswiderski) [inline jbpm-quarkus-example/pom.xml]: is this really used?

File: jbpm-quarkus-example/src/test/java/org/kie/kogito/examples/OrdersProcessTest.java [hunk 2]
```diff
@@ -21,49 +22,58 @@
...
-    
+
     @Inject
     @Named("demo.orderItems")
     Process<? extends Model> orderItemsProcess;
-    
+
+    @BeforeEach
+    public void setup() {
+        // abourt all intsances after each test
```
Comment (mswiderski) [inline jbpm-quarkus-example/src/test/java/org/kie/kogito/examples/OrdersProcessTest.java]: small typo in abort..

**Review by mswiderski — COMMENTED** [review]
overall looks good, just two small comments

**Comment by tsurdilo** [comment]
@mswiderski you are right about the depends - removed. Also updated rest test to not use hard-coded ids but get them from the post/get requests it does. thanks!

**Review by mswiderski — APPROVED** [review]

**Pull request merged**